terraform/install.sh

```diff
@@ -58,6 +60,51 @@ getBillingAccount() {
   log "using billing account: $billing_acct"
 }
 
+getProject() {
+  log "Checking for project list"
+  found_projects=$(gcloud projects list --filter="name:stackdriver-sandbox-*" --format="value(projectId)")
+  if [ -z "$found_projects" ] || [[ ${#found_projects[@]} -eq 0 ]]; then
+    create_flag=true
+  else
+      log "Which project would you like to use?:"
+      IFS_bak=$IFS
+      IFS=$'\n'
+      select opt in ${found_projects} "create" "cancel"; do
+        if [[ "${opt}" == "cancel" ]]; then
+          exit 0
+        elif [[ "${opt}" == "create" ]]; then
+          create_flag=true
+          log "create a new Sandbox!"
+          break
+        elif [[ -z "${opt}" ]]; then
+          log "invalid response"
+        else
+          project_id=$opt
+          bucket_name="$project_id-bucket"
+          break
+        fi
+      done
+      IFS=$IFS_bak
+  fi
+}
+
+createProject() {
+  if [ "$create_flag" = true ]; then
+    # generate random id
+    project_id="stackdriver-sandbox-$(od -N 4 -t uL -An /dev/urandom | tr -d " ")"
+    bucket_name="$project_id-bucket"
+    # create project
+    gcloud projects create "$project_id" --name="Stackdriver Sandbox Demo"
+    # link billing account
+    billing_id=$(gcloud beta billing accounts list --format="value(name)" --filter="displayName:$billing_acct")
```

Comment:
What happens if the user has two or more billing accounts?

Reply:
The filter is the account name user chose.